.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,26 @@
+
+name: terraform static-analysis test
```

Comment:
nit: The names of every workflow seem a little random:
- terraform static-analysis test
- k8s-manifest test
- docker-image scan test

can we make them a little more uniform?

Reply:
Agreed @imilchev. What do you think is the right naming convention?

Reply:
doesn't really matter... just that my OCD triggered here 😆 
- Terraform scanning
- K8s manifest scanning
- Docker image scanning

If someone has better suggestions, go ahead.

Reply:
👍